### PR TITLE
Rewrite of default mounted directories to be more consistent …/project; document in README

### PR DESCRIPTION
**Pull request by joesb**

### CODE REVIEW

File: config/config.yml [hunk 1]
```diff
@@ -62,7 +62,8 @@ build_composer: true
 drupal_composer_path: false
 drupal_composer_install_base_dir: "/var/www"
 drupal_composer_install_dir: "{{ drupal_composer_install_base_dir }}/drupalvm"
-drupal_composer_site_install_dir: "{{ drupal_composer_install_base_dir }}/sites"
+drupal_composer_site_host_dir: "~/Sites/projects"
+drupal_composer_site_install_dir: "{{ drupal_composer_install_base_dir }}/projects"
```
Comment (mikebell) [inline config/config.yml:66]: Shouldn't this just be {{ drupal_composer_install_base_dir }}? Default is now set to "~/Sites/projects/projects"

Comment (joesb) [reply config/config.yml:66]: `drupal_composer_install_base_dir` is used in both line 64 and 66 - need both variables

Comment (joesb) [reply config/config.yml:66]: … because two host directories are mounted 
1. the base directory of the Drupal VM;
2. the projects/sites directory

**Comment by joesb** [comment]
Are we cool to merge this now?

**Pull request merged**